build/config/compiler/BUILD.gn

```diff
@@ -105,6 +105,11 @@ config("abi_default") {
 }
 
 config("target_default") {
+  if (current_os == "tizen") {
+    import("//build_overrides/tizen.gni")
+    target_cflags = [ "--sysroot=$tizen_home" ]
```

Comment:
Also, this will cause an error if the target_cflags option is set. You want to append to `cflags` here not overwrite `target_cflags`.

Reply:
Done.